### PR TITLE
Update volto-slate configuration to indicate it is now part of the core.

### LINKED ISSUE #5342: [docs] configuration > volto-slate needs updates
[`volto-slate` is now in core](https://6.docs.plone.org/volto/upgrade-guide/index.html#volto-slate-is-now-in-core). The page [volto-slate](https://6.docs.plone.org/volto/configuration/volto-slate/index.html) is now out of date.

With the exception of Release Notes and the Upgrade Guide, anything that indicates `volto-slate` not being in core needs to be updated to it now being in core. Documentation was never updated when that happened back in v16.

No one has done a comprehensive search to identify files that need to be updated. It's not a simple find-and-replace. For example, on [volto-slate](https://6.docs.plone.org/volto/configuration/volto-slate/index.html):

> Although this add-on is in an early alpha stage

It's not in alpha any more. It was added in [Volto 16.0.0-alpha.15](https://6.docs.plone.org/volto/upgrade-guide/index.html#volto-slate-is-now-in-core). This page should have similar admonitions copied from the Upgrade Guide.

### PR DESCRIPTION
**Pull request by MostafaMagdyy**
Fixes #5342 for volto docs to indicate that volto-slate is now part of the core

### CODE REVIEW

**Comment by MostafaMagdyy** [comment]
Please review this pull request. @stevepiercy ,@polyester, could you please take a look and provide your feedback? Thank you!

**Comment by stevepiercy** [comment]
@MostafaMagdyy you have two pull requests open for the same issue (#5884). Neither will be reviewed until you close one. See Item 3 under [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do).

**Comment by MostafaMagdyy** [comment]
@stevepiercy Sorry for the inconvenience. I closed the first one because I did not pass the Plone contributor agreement.
Now you will find only one regarding this issue

File: docs/source/configuration/volto-slate/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ myst:
 
 # `volto-slate`
 
-`volto-slate` is an interactive default text editor for Volto, developed on top of {term}`Slate`.
+`volto-slate` is an interactive default text editor for Volto, integrated into the core system and developed on top of {term}`Slate`.
```
Comment (stevepiercy) [inline docs/source/configuration/volto-slate/index.md:14]: I think the more important clause should go first, although the lesser is a good addition for clarity.

```suggestion
`volto-slate` is an interactive default text editor for Volto, developed on top of {term}`Slate` and integrated into the core system.
```

File: packages/volto/news/5342.documentation [hunk 2]
```diff
@@ -0,0 +1 @@
+Update volto-slate configuration to indicate it is now part of the core. @MostafaMagdyy
```
Comment (stevepiercy) [inline packages/volto/news/5342.documentation:1]:
```suggestion
Update `volto-slate` configuration documentation to indicate it is now part of Volto core. @MostafaMagdyy
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
We also need to review and possibly rewrite all the files within the `configuration/volto-slate` directory. I will push some changes for that as well, because they require knowledge of its history.

**Comment by MostafaMagdyy** [comment]
@stevepiercy Thanks for your help regarding prioritizing the important clause. I have committed the suggested changes already. Please check if there are any other changes needed regarding this issue, **Also** I have reviewed the files within the configuration/volto-slate directory. and found these files focus on extending Volto-Slate into projects that use Volto, with guidance on creating customized plugins. So, I am not sure whether they need to be updated

**One thing I noticed that might require updating is this description is**
# `volto-slate` API

The `volto-slate` API provides methods to **integrate** the Slate editor into projects that use Plone's React-based frontend Volto.

I suggest it to be

The `volto-slate` API provides methods to **extends** the Slate editor into projects that use Plone's React-based frontend Volto.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@MostafaMagdyy it is actually an integration, not an extension, so we'll leave it as is.

I also reviewed the rest of the documentation in `volto-slate`, and it does not need any revisions.

This PR is OK to merge. Thank you!

**Comment by stevepiercy** [comment]
Actually, I want to make sure that all tests pass. For some reason "Acceptance Tests / Core Blocks - Listing (18.x) (pull_request)" did not pass. I restarted it to see if it is flaky and will pass this time.

**Pull request merged**